x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_table.test.tsx

```diff
@@ -140,4 +172,75 @@ describe('EnginesTable', () => {
       expect(tableContent).not.toContain('Universal');
     });
   });
+
+  describe('actions', () => {
+    it('when AppLogic.values.myRole.canManageEngines is false actions are hidden', () => {
```

Comment:
I generally prefer more meaningful, user oriented descriptions, but this is also fine if you prefer this

```suggestion
    it('will hide the action buttons if the user does not have permissions to manage this the engine', () => {
```

Reply:
0f879eb9604e780e066c1afc5bb072c1dee4cfc8